README.md

```diff
@@ -23,6 +23,78 @@ representation that uses an array to store the children of each node.
 However, for certain tasks it can be less performant, because some operations that modify
 the tree structure require iterating over all the children of a node.
 
+## Demo
+### Creating a Tree
+
+Can `addchild` or `addsibling`.
+```julia
+julia> using LeftChildRightSiblingTrees
+
+julia> mum = Node("Mum");
+
+julia> me = addchild(mum, "Me");
+
+julia> son = addchild(me, "Son");
+
+julia> daughter = addchild(me, "Daughter");
+
+julia> brother = addsibling(me, "Brother");
```

Comment:
Maybe move this up to right after the creation of `me`? Otherwise one might think you have 3 children, `son`, `daughter`, and `brother`.

Reply:
and/or a comment
```julia
# equivalent: addchild(mum, "Brother")
```

Reply:
I thought it was an interesting point to show I chould add a sibling after adding children.

Reply:
```suggestion
julia> brother = addsibling(me, "Brother");  # equivalent: to `addchild(mum, "Brother")`
```

Reply:
Sounds good. Then perhaps leave it there but add the comment?